### DISCUSSION TITLE
What exactly does "Safely save database files" do?

### DISCUSSION THREAD

**Discussion by notDavid** [keepassxreboot/keepassxc#6850] [Q&A] [answered]
`Settings > General > Safely save database files (disable if experiencing problems with Dropbox, etc.)`

What exactly is the difference/mechanism when enabling or disabling this setting?

Is it advised to enable this when using iCloud (or Dropbox, Google Cloud, etc.) ? does it have any side effects?

Also see ##5751

**Comment by droidmonkey** [comment, marked as answer]
See #1724

**Reply by notDavid** [reply]
Thank you for your reply @droidmonkey,

Okay i (kind of) understand that. So if i understand correct the default advise is just to keep this checkbox **enabled** when using iCloud sync… which should be safest.

**Reply by droidmonkey** [reply]
Correct
